lib/Dialect/FIRRTL/Transforms/IMConstProp.cpp

```diff
@@ -369,8 +369,16 @@ void IMConstPropPass::markBlockExecutable(Block *block) {
   for (auto &op : *block) {
 
     // Handle each of the special operations in the firrtl dialect.
-    if (isa<WireOp>(op) || isa<RegOp>(op))
-      markWireOrUnresetableRegOp(&op);
+    if (isa<WireOp>(op))
+      markWireOp(&op);
+    // TODO: RegOp and RegResetOp are conservatively marked overdefined to avoid
```

Comment:
I'd recommend dropping "TODO:".  If we change this in the future we won't need this TODO marker, and it is very possible it will never change.